### ISSUE TITLE
Compatibility with Adobe Scout

### ISSUE THREAD

**Issue by jcward** [jcward/hxtelemetry#7]
Feel free to voice your support for this issue here - it's on the back burner or now as it'll be a fairly difficult task. Plus I want to do some custom features between HxScout and hxtelemetry - more later.

**Comment by ncannasse** [comment]
What are the actual problems ?

**Comment by jcward** [comment]
@ncannasse, well, basically it's an investigation.

When I first tried to send data to Scout, Scout threw up an "Invalid telemetry data" message, or something. I tried to send [all the same headers](https://github.com/jcward/hxtelemetry/blob/master/src/hxtelemetry/HxTelemetry.hx#L99) I had dumped from FLM grabs of the players, but I couldn't make this message go away.

To make this easier to revisit later, I've noted a few places in the code where updates will be required for Scout compatibility, e.g. I don't have as much data as the actual FLM does - https://github.com/jcward/hxtelemetry/blob/master/src/hxtelemetry/HxTelemetry.hx#L232

I assume you're interested in this or you wouldn't have commented here?  =)

**Comment by ncannasse** [comment]
Well, I was curious what were the limitations you're having. I haven't yet tried your own Scout client but if it's as responsive and complete as the original Scout then I guess that's not a problem

**Comment by jcward** [comment]
Yeah, I mean it's probably doable, but it's not where I'm interested in spending time just yet (I need a clone or two!) I'd say HxScout is very responsive though not as complete yet, and there is some work to do on performance (particularly, some of the data merging could happen on a separate thread and update the UI.)  But overall I've very impressed with Haxe & OpenFL's performance for this app.

**Comment by jcward** [comment]
Getting further and further from this. The performance refactor (and the fact that the FLM flow is terribly slow) is probably the nail in the coffin of this issue.
